package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -304,14 +310,14 @@ class DiffusionMap(AnalysisBase):
         the collective coordinates.
     """
 
-    def __init__(self, DistanceMatrix, epsilon, weights=None, timescale=1):
+    def __init__(self, DistanceMatrix, Epsilon, weights=None, timescale=1):
```

Comment:
variable names are in **snake_case**. Only class names at definition are in *CamelCase*